### ISSUE TITLE
Data proprocessing

### ISSUE THREAD

**Issue by alrojo** [nyu-dl/dl4mt-cdec#2]
Thanks for the repo @jych!

Do you only apply tokenize and normalize-punctuation on the validation set and test set or also the training set?

Also, what dataset do you use for building the char_dictionary? Is it the training set or the valid/test sets as well?

**Comment by jych** [comment]
Hi @alrojo 

we apply same preprocessing to all available corpora.
We only use training corpus to build the char_dictionary.

**Comment by alrojo** [comment]
Thanks @jych!

One more question:
In [build_dictionary_char#L32](https://github.com/nyu-dl/dl4mt-cdec/blob/master/preprocess/build_dictionary_char.py#L32) you define 'unk' and 'eos', but what about start of sequence, also known as the 'go'/'sos' symbol? What is the first char you feed it on the decoder (at h_1).
